### PR TITLE
DM-34808: add nightly report mock up

### PR DESCRIPTION
**Pull request by SimonKrughoff**

### CODE REVIEW

File: nightly/auxtel.ipynb [hunk 1]
```diff
@@ -0,0 +1,465 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d05a6c33-82b1-47c0-9bb5-b856829982cf",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "repo = 'dp02'\n",
+    "butler = dafButler.Butler(repo, collections='2.2i/runs/DP0.2')\n",
```
Comment (jonathansick) [inline nightly/auxtel.ipynb:425]: We're running on `data-dev` at the moment, so I think we'll have to hold off until I'm deploying there and probably add a configuration field for the notebook sidecar YAML files to include/exclude specific RSP environments from syncing a given notebook.

**Review by jonathansick — COMMENTED** [review]
Looks good, and I just have some thoughts on simplifying the parameters to start off with.

Because it's using the Butler, this notebook apparently won't work in `data-dev`, where I'm initially running Times Square, so I'll have to start deploying Times Square more widely first before deploying this.

File: nightly/auxtel.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+title: AuxTel Nightly Report
+description: Nightly report for the AuxTel given an observation night
+authors:
+  - name: Simon Krughoff
+    slack: ksk
+tags:
+  - nightly
+  - auxtel
+parameters:
+  log_date:
```
[the remark this answers is not in the document]
Comment (SimonKrughoff) [reply nightly/auxtel.yaml:10]: Great suggestion.

File: nightly/auxtel.ipynb (hunk 1 above)
Comment (SimonKrughoff) [reply nightly/auxtel.ipynb:425]: I understand that we can't practice deployment until we have data at `data-dev` or we are deploying at `data-int`.

I'd like to close this PR for creating the first attempt.  I'll follow up on slack.

**Review by jonathansick — APPROVED** [review]
Ok, let’s go ahead with this and see how the system deals with the two issues:

- a parameter that needs heavy escaping (the URL)
- a notebook that crashes

**Pull request merged**